daemon/graphdriver/overlay/overlay.go

```diff
@@ -238,12 +255,32 @@ func (d *Driver) CreateReadWrite(id, parent string, opts *graphdriver.CreateOpts
 	return d.Create(id, parent, opts)
 }
 
+// Parse overlay storage options
+func (d *Driver) parseStorageOpt(storageOpt map[string]string, driver *Driver) error {
```

Comment:
This should be cleaned up.
I don't think we need to store stuff on a driver object, instead we can pass in an `overlayOptions` pointer.